spec/observables/dom/ajax-spec.ts

```diff
@@ -298,39 +294,41 @@ describe('ajax', () => {
 
     MockXMLHttpRequest.mostRecent.respondWith({
       status: 200,
-      contentType: 'application/json',
+      responseType: 'json',
       responseText: JSON.stringify(expected),
     });
 
     expect(result!.xhr).exist;
-    expect(result!.response).to.deep.equal(JSON.stringify({ foo: 'bar' }));
+    expect(result!.response).to.deep.equal({ foo: 'bar' });
     expect(complete).to.be.true;
   });
 
-  it('should fail if fails to parse response', () => {
+  it('should fail if fails to parse response in older IE', () => {
     let error: any;
     const obj: AjaxConfig = {
       url: '/flibbertyJibbet',
       responseType: 'json',
       method: '',
     };
 
+    // No `response` property on the object (for older IE).
+    MockXMLHttpRequest.noResponseProp = true;
```

Comment:
This is a new feature to the Mock that will strip the `response` property from the `xhr` to simulate an older browser that lacks that feature.